<launch>
    <node name="aircarROS" pkg="aircar_base" type="aircarROS" output="screen"/>

</launch>
